src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -496,8 +496,28 @@ private EventTypeStatistics validateStatisticsUpdate(
         if (existing != null && newStatistics == null) {
             return existing;
         }
-        if (!Objects.equals(existing, newStatistics)) {
-            throw new InvalidEventTypeException("default statistics must not be changed");
+        if (existing == null && newStatistics == null) {
```

Comment:
Maybe we can check if both are equal, if so, we don't need to validate again given it was validated before? So, the if would look like:
```
if ((existing == null && newStatistics == null) || existing.equals(newStatistics)) {
    return existing;
}
```